cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/vis_type_table/basic.spec.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  TABLE_INDEX_ID,
+  TABLE_INDEX_PATTERN,
+  TABLE_PATH_INDEX_DATA,
+  TABLE_INDEX_START_TIME,
+  TABLE_INDEX_END_TIME,
+  TABLE_CREATE_URL,
+  TABLE_VIS_APP_PATH,
+  TABLE_VIS_TYPE,
+} from '../../../../../utils/constants';
+
+/**
+ * Basic test suite description:
+ * Create a new table visualization from index pattern.
+ * Set the time range and Histogram Aggregation.
+ * Test that global header and side bar function correctly.
+ * Test that table visualization can be saved.
+ * Test inspector is enabled and contains correct data.
+ */
+
+describe('table visualization basic functions', () => {
+  before(() => {
+    cy.deleteIndex(TABLE_INDEX_ID);
+    cy.deleteIndexPattern(TABLE_INDEX_PATTERN);
+    cy.bulkUploadDocs(TABLE_PATH_INDEX_DATA);
+    cy.createIndexPattern(TABLE_INDEX_PATTERN, {
+      title: TABLE_INDEX_ID,
+      timeFieldName: 'timestamp',
+    });
+
+    cy.log('create a new table visualization: ', TABLE_CREATE_URL);
+    cy.visit(TABLE_CREATE_URL);
+
+    // Wait for page to load
+    cy.waitForLoader();
+
+    // Select index pattern and load table visualization
+    cy.setTopNavDate(TABLE_INDEX_START_TIME, TABLE_INDEX_END_TIME);
+    // Set a Histogram Aggregation
+    cy.addBucketsAggregation();
+    cy.splitRows();
+    cy.setupHistogramAggregation('age', '20', 2);
+  });
+
+  it('Should allow applying changed params', () => {
+    cy.setHistogramInterval('201', 2);
+    cy.isUpdateAggregationSettingsEnabled();
+  });
```

Comment:
Each test should be atomic. so that even if one test does not pass, the next test runs without being affected.

Reply:
Got it. I will modify this one. Thank you.